src/pm-conversations/PmConversationList.js

```diff
@@ -25,45 +27,48 @@ type Props = $ReadOnly<{|
 /**
  * A list describing all PM conversations.
  * */
-export default class PmConversationList extends PureComponent<Props> {
-  handleUserNarrow = (user: UserOrBot) => {
-    this.props.dispatch(doNarrow(pm1to1NarrowFromUser(user)));
+export default function PmConversationList(props: Props) {
+  const handleUserNarrow = (user: UserOrBot) => {
```

Comment:
Should use `useCallback` -- otherwise this is a new value each time this gets re-rendered, which in turn means each `UserItem` will get a new value and may have to re-render.

Reply:
Done.